static/docs/commands-reference/update.md

```diff
@@ -0,0 +1,30 @@
+# update
+
+Update dependencies and reproduce specified DVC-files.
+
+## Synopsis
+
+```usage
+usage: dvc update [-h] [-q | -v] targets [targets ...]
+
+positional arguments:
+  targets        DVC-files to update.
+```
+
+## Description
+
+After creating import stages with `dvc import`, the external data source can
+change. `dvc update` can be used to bring the imported <abbr>data
+artifact</abbr> up to date. This is currently the only method to do so.
+
+To indicate which import stage(s) to update, specify the corresponding
```

Comment:
can we remove this (s) plurals here?

Reply:
Done in 2cd68f6

You really hate "(s)"s right? Haha but you know they are everywhere in the docs right? 🙁  – should we make an issue for this?

Reply:
OK. I'll assume no issue needed here unless you let me know later. Thanks

Reply:
haha :) I don't know, they just look a bit artificial to me ... yeah, no need for a separate issue, we can remove them along the way